sdk/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -50,9 +50,14 @@ class TestTraceLog extends TraceLog {
   def getMessages: Seq[String] = messages.view.map(_._1).toSeq
 }
 
-class EvaluationOrderTest_V2 extends EvaluationOrderTest(LanguageVersion.v2_dev)
+class EvaluationOrderTest_V2 extends EvaluationOrderTest(LanguageVersion.v2_dev) {
+  protected def tuple2TyCon: String = {
```

Comment:
You could probably define this in the EvaluationOrderTest class by calling StablePackages(languageVersion.major).